Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts

```diff
@@ -24,7 +24,6 @@ const convertTimeIntervalToHoursMinutesAndSeconds =
 
 const calculateTimeUntil = (date: Date, unit: unitOfTime.Diff = 'milliseconds'):
     Duration => moment.duration(moment(date)
-    .utc(true)
```

Comment:
See other places where this is used like this and investigate if it could introduce some discrepancy if we remove utc(true) from here, but leave it in the other functions..

Reply:
The issue cannot be reproduced.